oak-segment-remote/src/main/java/org/apache/jackrabbit/oak/segment/remote/persistentcache/PersistentDiskCache.java

```diff
@@ -213,6 +219,23 @@ private void cleanUpInternal() {
         }
     }
 
+    private class SegmentCacheEntry {
```

Comment:
cosmetics: since this class is internal, we could have the members as `public final` and remove the getters to be more concise.